multimap/src/test/java/org/infinispan/multimap/impl/TxEmbeddedMultimapCacheTest.java

```diff
@@ -0,0 +1,181 @@
+package org.infinispan.multimap.impl;
+
+import static org.infinispan.functional.FunctionalTestUtils.await;
+import static org.infinispan.test.TestingUtil.extractComponent;
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertFalse;
+import static org.testng.AssertJUnit.assertTrue;
+
+import javax.transaction.NotSupportedException;
+import javax.transaction.SystemException;
+import javax.transaction.TransactionManager;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.multimap.EmbeddedMultimapCacheManagerFactory;
+import org.infinispan.multimap.api.MultimapCache;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.infinispan.util.concurrent.IsolationLevel;
+import org.infinispan.util.function.SerializablePredicate;
+import org.testng.annotations.Test;
+
+/**
+ * Multimap Cache with transactions in single cache
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 9.1
+ */
+@Test(groups = "functional", testName = "multimap.TxEmbeddedMultimapCacheTest")
+public class TxEmbeddedMultimapCacheTest extends EmbeddedMultimapCacheTest {
```

Comment:
Also if you can using the factory or @Factory annotation would be preferred over using different classes. Unless there are a lot of differences between how the tests run.